cli/tools/repl.rs

```diff
@@ -474,7 +443,8 @@ impl ReplSession {
     };
 
     // inject prelude
-    repl_session.evaluate_expression(PRELUDE).await?;
+    let prelude = include_str!("repl.js");
```

Comment:
I actually prefer it inline so that we don't have a js file in the tools directory. I don't think code formatting this is that important.

Reply:
> we don't have a js file in the tools directory

Why do you find that undesirable tho?

Reply:
Instead of this detail being buried it's presented at a higher level right inside the directory as if it were on that same level of importance and execution. I'd prefer it buried within the file because it's not that important and the code is not that large. If you really want this, I think it would be better to have a sub repl folder rather than both of these files being at the top level of tools. Sorry for being picky...

Reply:
That's alright, was just curious 🙂
As it is **right now** I don't care that much, I'll punt this until it grows.